### ISSUE TITLE
What is a development set here?

### ISSUE THREAD

**Issue by einareinarsson** [tensorflow/nmt#83]
Seq2seq is relatively new for me. Could someone kindly reveal what is a "development set" in this context? In my practice I used train, test, validation sets but this one is kind of a novelty for me...

**Comment by oahziur** [comment]
Hi @einareinarsson , We pick the model that performs best on the `dev set`, and then confirm the results with the `test set`.
